cms/articles/models.py

```diff
@@ -168,20 +168,17 @@ class StatisticalArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):  #
 
     content_panels: ClassVar[list["Panel"]] = [
         *BundledPageMixin.panels,
-        MultiFieldPanel(
-            [
-                TitleFieldPanel("title", help_text="Also known as the release edition. e.g. 'November 2024'."),
-                FieldPanel(
-                    "news_headline",
-                    help_text=(
-                        "Use this as a news headline. When populated, replaces the title displayed on the page. "
-                        "Note: the page slug is powered by the title field. "
-                        "You can change the slug in the 'Promote' tab."
-                    ),
-                    icon="news",
-                ),
-            ],
-            heading="Title",
+        TitleFieldPanel(
+            "title", placeholder="Edition*", help_text="Also known as the release edition. e.g. 'November 2024'."
```

Comment:
```suggestion
            "title",
            heading="Edition",
            placeholder="Edition*", 
            help_text="Also known as the release edition. e.g. 'November 2024'."
```

including `heading="Edition"` also updates it in the minimap

Reply:
added [2e7b3a3](https://github.com/ONSdigital/dis-wagtail/pull/276/commits/2e7b3a30dc5a09350cacc8dabe554db0633b3737)

Reply:
changed slightly in [b40732c](https://github.com/ONSdigital/dis-wagtail/pull/276/commits/b40732c0d029ac69fb98b269daa1d2091891d9c2)